src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/model/common/Event.java

```diff
@@ -17,4 +17,5 @@
     COMPLETED, // the processing of the envelope completed successfully
     // when envelope status needs to be updated for reprocessing (used manually to set the event with reason)
     MANUAL_STATUS_CHANGE,
+    ONLINE_STATUS_CHANGE,
```

Comment:
may be not just status change because status will change by system.
MANUAL_RETRIGGER_PROCESSING or sthelse